### PR TITLE
Restore navigation history top menu

### LINKED ISSUE #4668: Restore navigation history in top menu
### Description

Restore navigation history in top menu

### Acceptance criteria 

- Enable users to navigate back in history
- Ensure the [design](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=38%3A12121&t=4Sqe9TxP0GwPxahR-4) and functionality is restored
- Restore unit/e2e tests

### PR DESCRIPTION
**Pull request by oskarleonard**
### What was the problem?

This PR resolves #4668

### How was it solved?

By adding the navigation buttons back

### How was it tested?

Manually

### CODE REVIEW

**Comment by soroushm** [comment]
please adjust  width while user is logged in
Screenshot 2022-12-13 at 13 24 47

**Comment by soroushm** [comment]
Forward button working randomly

**Comment by oskarleonard** [comment]
@soroushm The border should be more pixel perfect now 👍 . I dont understand your second comment though, the navigation is the same as when it was removed.

File: src/modules/token/fungible/components/tokenAmount/index.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const TokenAmount = ({
   if (showInt) value = getInt(value);
   else if (showRounded) value = trim(value);
   return (
-    <Wrapper className={className}>
+    <Wrapper {...(className && { className })}>
```
Comment (soroushm) [inline src/modules/token/fungible/components/tokenAmount/index.js:42]: @oskarleonard what we gain from this condition? className are fine with null or undefiend

**Review by soroushm — DISMISSED** [review]

File: src/modules/token/fungible/components/tokenAmount/index.js (hunk 1 above)
Comment (oskarleonard) [reply src/modules/token/fungible/components/tokenAmount/index.js:42]: If we pass in React.Fragment as a wrapper react will complain if classname is a prop even if it is undefined. So this prevents that warning from happening.

**Review by soroushm — APPROVED** [review]

**Pull request merged**